src/handlers/localstorage/accountLocal.ts

```diff
@@ -195,6 +197,43 @@ export const savePurchaseTransactions = (
     purchaseTransactionsVersion
   );
 
+/**
+ * @desc get transactions
+ * @param  {String}   [address]
+ * @param  {String}   [network]
+ * @return {Object}
+ */
+export const getLocalPendingTransactions = (
+  accountAddress: any,
+  network: any
+) =>
```

Comment:
Out of sheer curiosity, why are you typing using a JSDoc and not just typescript types? Does it avoid type check but retains suggestions or something? I'm genuinely curious 😄

Reply:
tbh i was just keeping the file's patterns intact. I'm guessing this is undergoing TS refactor at some point, but didn't want to refactor it myself and include the unrelated diff.

Reply:
Makes sense, IIRC you could use JSDoc types on js files to get TS to work so it may be in flight, thanks for explanation I appreciate that!